docs/_docs/refs/mojito-locales.md

```diff
@@ -157,11 +164,11 @@ permalink: /docs/refs/mojito-locales/
 | uz-UZ	        | Uzbek (Uzbekistan) |
 | vi-VN	        | Vietnamese (Viet Nam) |
 | xh-ZA	        | Xhosa (South Africa) |
+| ypk-021	        | Yupik (Northern America) |
```

Comment:
yupik seems to be a group of language, https://en.wikipedia.org/wiki/Yupik_languages. `    "ypk": "ems ess esu ynk",` from this list only "esu" has a display name hence was generated from the script I wrote. All other entries you had are covered. 

Wondering if you really need this entry of if "esu" would work in your case.